tests/test_adapter.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from dataclasses import asdict
+import pytest
+
+
+@pytest.mark.parametrize("model_size", ["7B", "13B", "30B", "65B"])
+def test_config_identical(model_size, lit_llama):
```

Comment:
Test isn't doing much but it's honest work and will fail should the definitions get out of sync somehow